src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = days * 40;
+  const midDiscount = 20;
+  const largeDiscount = 50;
+  const longTerm = 7;
+  const midTerm = 3;
```

Comment:
More correctly to capitalize name of constants
```suggestion
  const MID_TERM = 3;
```